### DISCUSSION TITLE
Get Hex value from Filament Colors->primary?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#10282] [Help]
I use a Apex Charts Widget where i can set the colors. But the color is hardcoded. Is it possible to get the Hexcode for the primary color from Filamentcolors by method?

**Comment by leandrocfe** [comment]
You can use this `FilamentColor::getColors()['primary'][500]`
If you need to convert rgb to hex, you can use the [Spatie color](https://github.com/spatie/color)

**Comment by PetroZaburko** [comment]
@leandrocfe is right. 
In my case I created method:
```php
use Filament\Support\Facades\FilamentColor;
use Spatie\Color\Rgb;

protected function getColor(): string
{
        $rgb = FilamentColor::getColors()['primary'][500];
        return Rgb::fromString("rgb($rgb)")->toHex();
}
```
and call it in `protected function getOptions(): array` like:
```php
'colors' => [$this->getColor()],
```
